rllib/evaluation/tests/test_trajectory_view_api.py

```diff
@@ -188,9 +187,14 @@ def test_traj_view_next_action(self):
         )
         # Add the next action to the view reqs of the policy.
         # This should be visible then in postprocessing and train batches.
+        # Switch off for action computations (can't be there as we don't know
+        # the next action already at action computation time).
         rollout_worker_w_api.policy_map[DEFAULT_POLICY_ID].view_requirements[
             "next_actions"] = ViewRequirement(
-                SampleBatch.ACTIONS, shift=1, space=action_space)
+                SampleBatch.ACTIONS,
+                shift=1,
+                space=action_space,
+                used_for_compute_actions=False)
```

Comment:
maybe we should have validation for this field somewhere? seems easy to miss, and not straight-forward for regular users.

Reply:
You mean to check, whether it's even possible to have this in the action computation event, even though the shift is >0 from a "collected" field, like actions? Great idea!